tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py

```diff
@@ -10,54 +12,128 @@ def __init__(self, *args, **kwargs):
 
     def setUp(self):
         super(BasicTests, self).setUp()
-        self.openvcloud = self.random_string()
-        self.vdcusers = [{'kheirj': {'openvcloud': self.openvcloud,
-                                     'provider': 'itsyouonline',
-                                     'email': 'kheirj@greenitglobe.com'}}]
         self.acc1 = self.random_string()
         self.cs1 = self.random_string()
-        self.accounts = [{self.acc1: {'openvcloud': self.openvcloud}}]
-        self.cloudspaces = [{self.cs1: {'account': self.acc1}}]
+        self.vdcuser = self.random_string()
+        self.vdcusers[self.vdcuser] = {'openvcloud': self.openvcloud,
+                                       'provider': 'itsyouonline',
+                                       'email': '%s@test.com' % self.random_string(),
+                                       'groups': ['user']}
+        self.accounts = {self.acc1: {'openvcloud': self.openvcloud}}
+        self.cloudspaces = dict()
+
+        self.temp_actions = {'account': {'actions': ['install']},
+                             'vdcuser': {'actions': ['install']},
+                             'vdc': {'actions': ['install']}
+                            }
+
 
-    def test001_trial(self):
+    @unittest.skip('https://github.com/openvcloud/0-templates/issues/47')
+    def test001_create_cloudspace_with_wrong_params(self):
         """ ZRT-OVC-001
-        *Test case for ...*
+        *Test case for creating acloudspace with wrong parameters .*
 
         **Test Scenario:**
+        #. Create an cloudspace with providing non existing parameter, should fail.
```

Comment:
Please provide this step.. (check the accounts file for that)
- Create an account with providing non existing user, should fail